jahorwitz/slopopedia
# Feature/SLOP-68/Create Sounds Page

## Describe your changes

I created the slop-card and slop-card-list components. The slops-route was updated to use these components. Sound context was used from Fred's "I'm Feeling Sloppy" functionality to avoid future merge conflicts. Sound context allows audio to be played.

**How to test**
Add a sound from Slopsounds.csv to your database (copy the URL with .m4a extension)
Image shouldn't matter (purple goblin is default if you don't upload an image)
Launch the back end and front end, and visit the site
Click the main page banner
Sounds should populate (just images), clicking them should play a sound

## Issue ticket number and link

[Slop-68](https://tripleten-apiary.atlassian.net/jira/software/projects/SLOP/boards/2?selectedIssue=SLOP-68)

## Checklist before requesting a review

- [ ] If there are changes to components, I have added / updated automated tests
- [ ] I have made any necessary updates to Storybook to accompany these changes
